### PR TITLE
Build: fix prettydocs CI flakyness

### PR DESCRIPTION
**Pull request by ndelangen**
Issue:

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by ndelangen** [comment]
@shilman self-merging this

@kasperpeulen this should fix the inconsistent CI job, I think

File: .circleci/config.yml [hunk 1]
```diff
@@ -637,3 +646,5 @@ workflows:
           parallelism: 26
           requires:
             - build-sandboxes
+
+# VS Code Extension Version: 1.5.0
```
Comment (yannbf) [inline .circleci/config.yml:650]: Was this line intentional @ndelangen ?

Comment (ndelangen) [reply .circleci/config.yml:650]: no!, how did this happen?

Comment (ndelangen) [reply .circleci/config.yml:650]: fixed here:
https://github.com/storybookjs/storybook/pull/20510

**Pull request merged**